### DISCUSSION TITLE
What are the expected performance results?

### DISCUSSION THREAD

**Discussion by aguirrejuan** [weecology/DeepForest#568] [Q&A]
Hi, 

Can you estimate what the expected performance results are? Or the different results that the model obtained in the pre-training phase in terms of box recall and precision?

Thanks

**Comment by ethanwhite** [comment]
We have a couple of papers that cover the performance on National Ecological Observatory Network data. Probably the best starting point is [A remote sensing derived data set of 100 million individual tree crowns for the National Ecological Observatory Network](https://doi.org/10.7554/eLife.62922) and in particular the [Evaluation and validation](https://elifesciences.org/articles/62922#s3-2) section. Take a look at that and let us know if you have any follow up questions.

**Reply by aguirrejuan** [reply]
Thank you very much.
